pkg/menu/menu_test.go

```diff
@@ -367,16 +367,6 @@ func TestDisplayMenu(t *testing.T) {
 				pressKey(uiEvents, []string{"2", "a", "<Backspace>", "<Enter>"})
 			},
 		},
-		{
```

Comment:
I'd prefer that we comment this out, rather than remove it, because I'd like this test to work some day.